### DISCUSSION TITLE
static link of the C-interface of DeepMD libraries

### DISCUSSION THREAD

**Discussion by taccchen** [deepmodeling/deepmd-kit#4129] [General]
Dear DeepMD developers, I am a new DeepMD user. While I can successfully build "libdeepmd_c.so" using the following cmake commands: " cmake -DUSE_TF_PYTHON_LIBS=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/DEEPMD2211 -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ ..; make; make install", I am wondering if it is possible to build static DeepMD libraries such as "libdeepmd_c.a". Thanks. Hanning

**Comment by njzjz** [comment]
There is no such the option.
